stable/yugabyte/generate_kubeconfig.py

```diff
@@ -63,7 +63,7 @@ def run_command(command_args, namespace=None, as_json=True):
                        '--certificate-authority={}'.format(ca_crt_file.name)]
     run_command(set_cluster_cmd, as_json=False)
 
-user_token = base64.b64decode(secret_data['data']['token'])
```

Comment:
we also do this at line 55. The best option would be to make run_command return strings all the time instead of bytes in python3 and strings in python3.

Maybe run it against python2 too to test that doing a decode('utf-8') returns the expected results.

Reply:
@iSignal L55 actually does require a byte string since the default mode for `tempfile.NamedTemporaryFile()` is actually 'w+b'. I tested that it generates a file and eyeballed it afterwards on py2.

Reply:
oh ok, didn't realise that! I'd argue that converting it all out to string makes for less subtle code and is easier to understand.